docs/visualize/lens.asciidoc

```diff
@@ -0,0 +1,143 @@
+
+Line:: Display data trends over time. In line charts, data is displayed as a
+series of data points that are connected by line segments.
++
+[role="screenshot"]
+image::images/lens_line_chart.png[Lens line chart]
+
+Area:: Display one or more quantities over time. In area charts, points are
+plotted and connected by lines to show the value of a quantity at several
+different times.
++
+[role="screenshot"]
+image::images/lens_area.png[Lens area chart]
+
+Stacked Area:: Display the evolution of numeric variables for several
+categories stacked on top of each other.
++
+[role="screenshot"]
+image::images/lens_stacked_area.png[Lens stacked area chart]
+
+Data Table:: Display your raw data or aggregation results in a tabular format.
++
+[role="screenshot"]
+image::images/lens_data_table.png[Lens data table]
+
+Metric:: Display the latest number in a data series.
++
+[role="screenshot"]
+image::images/lens_metric.png[Lens metric]
+
+[float]
+[[create-lens-visualization]]
+=== Create Lens visualizations
+
+To create your Lens visualizations, use the visualization builder.
+
+[float]
+[[change-index]]
+==== Change the index
+
+If you change your mind about the index pattern that you want to use, change your index right from the visualization builder.
+
+To change the index, click *change* next to the index name, then select the index.
+[role="screenshot"]
+image::images/lens_change_index.gif[Change index]
+
+[float]
+[[choose-your-data]]
+==== Choose the data you want to display
+
+The available data fields appear in the editor on the left.
+
+If you want to see the top values of a field, click *i* next to the field.
+[role="screenshot"]
+image::images/lens_top_values.gif[Top values]
+
+To narrow the data fields that are displayed:
+
+* Enter the field name in *Search for fields*.
+
+* Click *Fields filtered*, then click the filter type.
++
+[role="screenshot"]
+image::images/lens_fields_filtered.png[Fields filtered]
+
+[float]
+[[add-data]]
+==== Add the data you want to display
+
+To add data to the Lens visualization builder, drag and drop the data fields to one of these locations:
+
+* *Create a visualization* window
++
+[role="screenshot"]
+image::images/lens_data_fields.gif[Data fields]
+
+* *X-axis* and *Y-axis* fields
++
+[role="screenshot"]
+image::images/lens_data_fields2.gif[Data fields]
+
+[float]
+[[change-visualization-type]]
+==== Change the visualization type
+
+If you want to display your data in a different visualization, use one of the following options:
+
+* Next to the visualization type, click *change*, then select the visualization.
++
+[role="screenshot"]
+image::images/lens_change_visualization.gif[Visualization types]
+
+* Click the *Suggestions*
++
+[role="screenshot"]
+image::images/lens_suggestions.gif[Suggestions]
+
+[float]
+[[lens-next-steps]]
+==== Next steps
+
+When you've finished making changes to the visualization, *Save* it, then add your Lens visualizations to:
```

Comment:
We might need to update the Discover and Dashboard docs to mention Lens.

Reply:
I can't figure out a way to add Lens visualization to Discover, so I think the document I pulled this from was incorrect. 

Also, I we won't need to add Lens to the Dashboard section because you access Lens from Visualize, and the Dashboard section already covers that info.